test_density/setup_emap.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+# encoding: utf-8
+import numpy as np
+import meld
+from meld.comm import MPICommunicator
+from meld import comm, vault
+import mdtraj as md
+from meld import system
+from meld.remd import ladder, adaptor, leader
+from meld.system.scalers import LinearRamp,ConstantRamp
```

Comment:
Almost all of these should now be improted into the base `meld` module. So you should only need `import meld` and them `meld.LinaerRamp`, etc. It's not a big deal, but it cleans up the imports a lot.